src/components/homepage/HeroSection/HeroDropdownSchema.jsx

```diff
@@ -0,0 +1,34 @@
+import * as Yup from "yup"
+
+import {
+  DROPDOWNELEM_TITLE_MIN_LENGTH,
+  DROPDOWNELEM_TITLE_MAX_LENGTH,
+  HERO_DROPDOWN_TITLE_MIN_LENGTH,
+  HERO_DROPDOWN_TITLE_MAX_LENGTH,
+} from "utils/validators"
+
+export const HeroDropdownOptionSchema = Yup.object({
+  title: Yup.string()
+    .min(
+      DROPDOWNELEM_TITLE_MIN_LENGTH,
+      `Title must be longer than ${DROPDOWNELEM_TITLE_MIN_LENGTH} characters`
+    )
+    .max(
+      DROPDOWNELEM_TITLE_MAX_LENGTH,
+      `Title must be shorter than ${DROPDOWNELEM_TITLE_MAX_LENGTH} characters`
+    ),
+  url: Yup.string(),
```

Comment:
same as before, we could either use `Yup.string.url()` or write our own

Reply:
842faf2f04e

Reply:
https://github.com/isomerpages/isomercms-frontend/commit/842faf2f04e8374d4cd438d296525ae26fbc8283